cypress/e2e/clue/branch/student_tests/dataflow_tool_spec.js

```diff
@@ -11,10 +11,11 @@ context('Dataflow Tool Tile', function () {
     cy.clearQAData('all');
     cy.visit(queryParams);
     cy.waitForLoad();
-    cy.collapseResourceTabs();
   });
   describe("Dataflow Tool", () => {
     it("renders dataflow tool tile", () => {
+      cy.wait(120000);
```

Comment:
2 minute wait on every run 😱 

I know this test has caused a lot of flakiness, so maybe it's best to be super conservative and make sure it has a chance to load. But why does it take 2 minutes to load? To me, this feels like a temporary work around that doesn't address the real underlying problem, though I'll freely admit that I don't know what the real problem is or how to fix it.

Reply:
For some reason, the default problem document in the `dfe` unit takes really long to load in github (This doesn't happen in real life and it's also not all that bad in a local cypress test) - and almost always it is the initial load time of the default problem document that has caused the test to fail. I tried a 1 min timeout and that didn't improve things. So, with this PR, I'm trying two things: 
- moving the check to see if the document loads into the test itself from the `before` hook so that it gets retries when it fails (Cypress doesn't retry code that fails in the `before` and `after` hooks)
- increasing the load time to 2 min. It's not ideal, but it is only for this one test and not everywhere and since this test runs in parallel with other ones, my hope is that this wait will not much affect the overall test running times. I've also increased the number of threads that the test job uses from 5 to 7 which has improved the average test run times from  30 min to 20 min.